src/components/calcite-combobox-item/calcite-combobox-item.tsx

```diff
@@ -58,6 +58,9 @@ export class CalciteComboboxItem {
   /** A unique value used to identify this item - similar to the value attribute on an <input>. */
   @Prop({ reflect: true }) value!: string;
 
+  /** Don't filter this item based on the search text */
+  @Prop({ reflect: true }) alwaysVisible: boolean;
```

Comment:
nitpick: maybe we can think of a better name? How about `constant`, `keep`, `hold`?

Reply:
`constant` is pretty good